tests/swig.at

```diff
@@ -0,0 +1,69 @@
+# Souffle - A Datalog Compiler
+# Copyright (c) 2019, The Souffle Developers. All rights reserved
+# Licensed under the Universal Permissive License v 1.0 as shown at:
+# - https://opensource.org/licenses/UPL
+# - <souffle root>/licenses/SOUFFLE-UPL.txt
+
+m4_define([SWIG_LANGUAGES], [[java],
+  [python]])
+
+dnl Run driver program for given language to output csv files
+dnl $1 current testsuite directory
+dnl $2 language 
+m4_define([LANGUAGE_TEST],[
+  m4_if($2, java, [
+    javac *.java
+    java -Djava.library.path=$1 driver
+  ])
+  
+  m4_if($2, python, [
+    python3 driver.py

```

Comment:
This should be wrapped in AT_CHECK so it's actually a check
```suggestion
    AT_CHECK([python3 driver.py], [0])
```

Reply:
fixed